gateway/webhook/webhookTransformer.go

```diff
@@ -63,6 +60,8 @@ func (bt *batchWebhookTransformerT) transform(events [][]byte, sourceType string
 	bt.stats.transformTimerStat.End()
 	if err != nil {
 		err := fmt.Errorf("JS HTTP connection error to source transformer: URL: %v Error: %+v", url, err)
+		statusCode := http.StatusServiceUnavailable
+		countWebhookErrors(sourceType, statusCode, len(events))
```

Comment:
I believe it would be better for this counter to be incremented by only one method, during `batchWebhookTransformerT#batchTransformLoop`.

It is hard to reason now whether the current implementation is correct and doesn't miss or count more stats than it should.

Reply:
Yes I missed this, moved metrics counting to batchWebhookTransformerT#batchTransformLoop